packages/clarity-devtools/src/config.ts

```diff
@@ -0,0 +1,16 @@
+import { Core, Data, Layout } from "clarity-js";
+
+export default function(): Core.Config {
+    return {
+        regions: [
+            [Data.Region.Header, "header", 1, Layout.Constant.DevHook], /* 1: Javascript Filter */
+            [Data.Region.Footer, "footer", 0, "product"], /* 0: Url */
+            [Data.Region.Navigation, "nav"]
+        ],
```

Comment:
What's this file for?

Reply:
For dev tools that we package with clarity to test out region functionality.